lib/apartment/adapters/abstract_adapter.rb

```diff
@@ -169,18 +170,17 @@ def connect_to_new(tenant)
       #   @return {String} tenant name with Rails environment *optionally* prepended
       #
       def environmentify(tenant)
-        unless tenant.include?(Rails.env)
-          if Apartment.prepend_environment
-            "#{Rails.env}_#{tenant}"
-          elsif Apartment.append_environment
-            "#{tenant}_#{Rails.env}"
-          else
-            tenant
-          end
-        else
+        if tenant.include?(Rails.env)
           tenant
+        else
+          [Apartment.db_prefix, tenant, Apartment.db_suffix].compact.join('_')
         end
       end
+      deprecate :environmentify, :internal_name, 2019, 8
+
+      def internal_name(tenant)
```

Comment:
This method is currently not in use, but extracted for further refactoring without altering original behavior